### DISCUSSION TITLE
How do I remove the red trail behing the cursor while using this. It bothers me A LOT.

### DISCUSSION THREAD

**Discussion by Rozevskis** [mayank1513/vscode-extension-trello-kanban-board#45] [Q&A] [answered]
How do I remove the red trail behind the cursor while using this. It bothers me A LOT.

**Comment by mayank1513** [comment]
Thanks for reaching out. Will fix this ASAP.

**Comment by Rozevskis** [comment]
![image](https://github.com/mayank1513/vscode-extension-trello-kanban-board/assets/44755996/055e8ad4-f929-4689-a998-d267152ea27b)
Why is it like this now :( I just want a button to disable the trail

**Comment by mayank1513** [comment, marked as answer]
Please open the drawer and disable the trails.
